code/modules/surgery/blood_filter.dm

```diff
@@ -31,7 +31,10 @@
 /datum/surgery_step/filter_blood/success(mob/user, mob/living/carbon/target, target_zone, obj/item/tool, datum/surgery/surgery, default_display_results = FALSE)
 	if(target.reagents?.total_volume)
 		for(var/datum/reagent/chem as anything in target.reagents.reagent_list)
-			target.reagents.remove_reagent(chem.type, min(chem.volume * 0.22, 10))
+			//SKYRAT EDIT BEGIN - Blood filtering no longer filters out formaldehyde
+			if(!istype(chem, /datum/reagent/toxin/formaldehyde))
```

Comment:
Making this check for some new variable/flag on the chem, instead of specifying one chem, might be a good idea for the sake of making it easier for more chems to do this in the future

Reply:
no, that would be bad, adding a new variable just for something like this isn't a good idea, make it a static list instead